webpack/assets/javascripts/react_app/components/SearchButton/SearchButton.js

```diff
@@ -0,0 +1,19 @@
+import URI from 'urijs';
+import React from 'react';
+import { Button, Icon } from 'patternfly-react';
+
+const handleClick = (searchQuery) => {
+  const uri = new URI(window.location.href);
+  uri.setSearch('search', searchQuery.trim());
+  window.Turbolinks.visit(uri.toString());
```

Comment:
don't we have duplication here with https://github.com/theforeman/foreman/pull/5771/files#diff-ccc2dd1b48def42815b6974e5a635e4a? and tfm.tools...